index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta content="width=device-width">
+    <title>Bacon &amp; Eggs</title>
+    <link rel="stylesheet" href="./css/style.css" />
+  </head>
+  <body>
+    <main>
+      <article class="post">
+        <header class="post__header">
+          <div class="post__headerLogo">Bacon &amp; Eggs</div>
+          <button class="post__headerButton">Learn More</button>
+          <nav>
+            <ul class="post__headerNav">
+              <li class="first"><a href="#">June 30, 2015</a></li>
+              <li>&bull;</li>
+              <li><a href="#">Poe</a></li>
+              <li>&bull;</li>
```

Comment:
Consider using pseudo-selectors with the `content` attribute to create this instead